dbt_platform_helper/providers/secrets.py

```diff
@@ -83,3 +79,35 @@ def get_parameter_name(self, addon_type: str, addon_name: str, access: str) -> s
 
     def _normalise_secret_name(self, addon_name: str) -> str:
         return addon_name.replace("-", "_").upper()
+
+
+class AddonNotFoundException(AWSException):
+    def __init__(self, addon_name: str):
+        super().__init__(f"""Addon "{addon_name}" does not exist.""")
+
+
+class AddonTypeMissingFromConfigException(AWSException):
+    def __init__(self, addon_name: str):
+        super().__init__(
+            f"""The configuration for the addon {addon_name}, is misconfigured and missing the addon type."""
+        )
+
+
+class InvalidAddonTypeException(AWSException):
+    def __init__(self, addon_type):
+        self.addon_type = addon_type
+        super().__init__(
+            f"""Addon type "{self.addon_type}" is not supported, we support: {", ".join(CONDUIT_ADDON_TYPES)}."""
+        )
+
```

Comment:
The above 3 should perhaps inherit from PlatformException. They aren't errors in AWS, but errors in our config (or perhaps that our config hasn't yet been run in)

Reply:
Added a couple of base classes and Todos for a subsequent refactor.